pkg/webhook/pod/pods.go

```diff
@@ -78,7 +86,21 @@ func (pc *PodAdmissionControl) AdmitPods(ar v1beta1.AdmissionReview) *v1beta1.Ad
 	name := ar.Request.Name
 	namespace := ar.Request.Namespace
 	operation := ar.Request.Operation
-	klog.Infof("receive admission to %s pod[%s/%s]", operation, namespace, name)
+	serviceAccount := ar.Request.UserInfo.Username
+	klog.Infof("receive %s pod[%s/%s] by sa[%s]", operation, namespace, name, serviceAccount)
+
+	isUnknownServiceAccounts := true
+	for _, sa := range pc.serviceAccounts {
```

Comment:
Suggest converting the SAs to a map[string]struct{}.

Reply:
The `pc. serviceAccounts` was a string list and I don't think it's necessary to fetch serviceAccounts from apiServer here. The comparison of string is enough.

Reply:
fix